portfolio/src/main/webapp/style.css

```diff
@@ -105,7 +111,7 @@
   color: white; 
```

Comment:
Let's have these lines sorted.

Reference:
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
Here and elsewhere.

Reply:
Updated.